airflow/executors/kubernetes_executor.py

```diff
@@ -737,7 +737,7 @@ def _adopt_completed_pods(self, kube_client: client.CoreV1Api) -> None:
                 kube_client.patch_namespaced_pod(
                     name=pod.metadata.name,
                     namespace=pod.metadata.namespace,
-                    body=PodGenerator.serialize_pod(pod),
+                    body={"metadata": {"labels": {"airflow-worker": pod.metadata.labels['airflow-worker']}}},
```

Comment:
Maybe we shouldn't mutate `pod` above and either use a variable or call `make_safe_label_value` here?